x-pack/platform/plugins/shared/cases/public/components/markdown_editor/eui_form.tsx

```diff
@@ -67,6 +69,17 @@ export const MarkdownEditorForm = React.memo(
       const conflictWarningText = i18n.VERSION_CONFLICT_WARNING(
         id === 'description' ? id : 'comment'
       );
+      const { services } = useKibana();
```

Comment:
Could you please move this logic into the `usePlugins` hook that is in `x-pack/platform/plugins/shared/cases/public/components/markdown_editor/use_plugins.ts` instead?

Reply:
Oh... this is a great idea! 
How does it look? https://github.com/elastic/kibana/pull/218089/commits/9127c28e98610698d1f31fbd0614e56e0ba5af85

Reply:
LGTM!